Makefile

```diff
@@ -11,8 +11,14 @@ lint:
 .PHONY: unit-test
 unit-test:
 	-docker run -d -p 5432:5432 --name test_db -e POSTGRES_PASSWORD=test_pw postgres
-	PYTHONWARNINGS=ignore:ResourceWarning python3 -m coverage run \
+	-PYTHONWARNINGS=ignore:ResourceWarning python3 -m coverage run \
 		-m unittest discover --start-directory tests/unit/backend --top-level-directory . --verbose
+	-make clean_test_db
```

Comment:
I think you're supposed to use `$(MAKE)` for reasons, see https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html

And `clean_test_db` is also `PHONY` I think.

But I think the bigger issue is that `make unit-test` will have a zero exit code even if the test fails, which will make the tests green in a lot of frameworks.

Reply:
Maybe you can do like a
```
python3 -m ... ; test_result=$$?; $(MAKE) clean_test_db; exit $$test_result
```
or something like that.

Reply:
TIL. Thanks for the tips - I've switched over to the commands you suggested.